bundles/org.openhab.binding.tellstick/src/main/java/org/openhab/binding/tellstick/internal/live/TelldusLiveDeviceController.java

```diff
@@ -71,7 +71,7 @@ public class TelldusLiveDeviceController implements DeviceChangeListener, Sensor
     public static final long DEFAULT_INTERVAL_BETWEEN_SEND = 250;
     private static final int REQUEST_TIMEOUT_MS = 15000;
     private AsyncHttpClient client;
-    static final String HTTP_API_TELLDUS_COM_XML = "http://api.telldus.com/xml/";
+    static final String HTTP_API_TELLDUS_COM_XML = "http://pa-api.telldus.com/xml/"; // http://api.telldus.com/xml/"
```

Comment:
It's the responsibility of the version control system to keep track of historic code not the code itself:

```suggestion
    static final String HTTP_API_TELLDUS_COM_XML = "http://pa-api.telldus.com/xml/";
```